HymnsTests/Earl Grey Tests/EarlGreyUITests.swift

```diff
@@ -0,0 +1,105 @@
+import XCTest
+import EarlGrey
+@testable import Hymns
+
+class EarlGreyUITests: XCTestCase {
+
+    override func setUp() {
```

Comment:
Can you make a new target for these? So that way we have some separation between unit tests and ui test?

Reply:
Yeah, good idea. I am calling it functionalTesting because I noticed that is what a lot of other people call this kind of thing like KIF which is a similar type of UI testing using the unit testing from Xcode